packages/celotool/register.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
It would be much better to make these `ts` commands as part of the celotool CLI instead of scripts.